codes/2_analyzing.py

```
@@ -105,51 +107,50 @@ def get_write_msg(todo_file_name, todo_file_desc):
-        
-    completion = api_call(trajectories)
-    
-    # response
-    completion_json = json.loads(completion.model_dump_json())
-    responses.append(completion_json)
+        print(f"[ANALYSIS] {todo_file_name}")
+        if todo_file_name == "config.yaml":
+            continue
+
+        if todo_file_name not in logic_analysis_dict:
+            # print(f"[DEBUG ANALYSIS] {paper_name} {todo_file_name} is not exist in the logic analysis")
```

Pylint (3.3.7)

[convention] Line too long (105/100)

(C0301)